### PR TITLE
Modal docs

### PR DESCRIPTION
**Pull request by AnnaRybkina**
image

Some styles are updated here as it was needed after the uno upgrade. We need to figure out why the tokens for the font sizes do not work after latest changes

### CODE REVIEW

**Review by CarolineGard — COMMENTED** [review]
Nice work! 

It looks a bit weird for me when running tech-docs here, also after npm i 
Screenshot 2023-05-24 at 15 15 12

**Review by CarolineGard — COMMENTED** [review]
Nice work! 

It looks a bit weird for me when running tech-docs here, also after npm i 
Screenshot 2023-05-24 at 15 15 12

**Comment by AnnaRybkina** [comment]
> Nice work!
> 
> It looks a bit weird for me when running tech-docs here, also after npm i Screenshot 2023-05-24 at 15 15 12

I've retested locally and I can't see any problems 🤷 Maybe you can kill node_modules and/or pnpm-lock?

File: docs/.vitepress/theme/custom.css [hunk 1]
```diff
@@ -1,6 +1,5 @@
 @font-face{font-family:Finntype;font-style:normal;font-weight:400;src:url("https://static.finncdn.no/_c/static/fonts/FINNTypeStrippet-Light.woff2") format("woff2")}
 @font-face{font-family:Finntype;font-style:normal;font-weight:700;src:url("https://static.finncdn.no/_c/static/fonts/FINNTypeStrippet-Medium.woff2") format("woff2")}
-@unocss-placeholder
```
Comment (AnnaRybkina) [inline docs/.vitepress/theme/custom.css:3]: This should not be here

File: docs/.vitepress/theme/custom.css [hunk 2]
```diff
@@ -9,6 +8,9 @@
   --vp-c-brand-light: #5e9cfd;
   --vp-c-brand-lighter: #9ac1fe;
 }
+html {
+  font-size: 62.5%;
```
Comment (AnnaRybkina) [inline docs/.vitepress/theme/custom.css:12]: Needed to make the rem based styles work in shadow dom

**Review by Skadefryd — APPROVED** [review]
Looks amaZing!

**Pull request merged**